HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Details_page.Steps.cs

```diff
@@ -0,0 +1,112 @@
+using Xunit;
+using Xunit.Abstractions;
+
+namespace HealthMonitoring.AcceptanceTests.Scenarios.Selenium
+{
+    public partial class Details_page : FeatureFixture, IDisposable
+    {
+        #region Private vars
+
+        private readonly string _rowsWithTags =
+            $"//table[contains(@class,'endpoints')]//tr/td[6]//span[text() = '{SeleniumHelper.UniqueTags[0]}']//ancestor::tr";
+        private readonly string _endpointLinksOfTagRowsOnHomePage;
+        private readonly string _endpointGroupsOfTagRowsOnHomePage;
+        private readonly string _endpointTagsOnHomePage;
+
+        private const string _endpointNameOnDetailsPage = "/html/body/table[1]/tbody/tr[3]/td[2]";
+        private const string _endpointGroupOnDetailsPage = "/html/body/table[1]/tbody/tr[2]/td[2]";
+        private const string _endpointTagsDetailsPage = "/html/body/table[1]/tbody/tr[9]/td[2]//span";
+
+        private readonly IWebDriver _driver;
+        private readonly RestClient _client;
+        private string _endpointName;
+        private string _endpointGroup;
+        private List<string> _edpointTags;
+        private readonly string _homePageUrl = $"{SeleniumConfiguration.BaseUrl}?endpoint-frequency=1000000&config-frequency=1000000";
+        #endregion
+
+        public Details_page(ITestOutputHelper output) : base(output)
+        {
+            _client = ClientHelper.Build();
+            _driver = SeleniumConfiguration.GetWebDriver();
+            _client.RegisterTestEndpoints();
+
+            _endpointLinksOfTagRowsOnHomePage = $"{_rowsWithTags}//td[2]//a";
+            _endpointGroupsOfTagRowsOnHomePage = $"{_rowsWithTags}//td[1]";
+            _endpointTagsOnHomePage = $"{_rowsWithTags}//td[6]//span";
+        }
+
+        public void Given_home_page()
+        {
+            _driver.LoadUrl(_homePageUrl);
+        }
+
+        public void When_user_clicked_on_endpoint_name()
+        {
+            var detailsLink = _driver.FindElements(By.XPath(_endpointLinksOfTagRowsOnHomePage)).First();
+            _endpointName = detailsLink.Text;
+            _endpointGroup = _driver.FindElements(By.XPath(_endpointGroupsOfTagRowsOnHomePage)).First().Text;
+            _edpointTags = GetTagsOfFirstEndpointOnHomePage();
+
+            _driver.LoadUrl(detailsLink.GetAttribute("href"));
+        }
+
+        public void Then_name_and_group_should_be_the_same_as_on_home_page()
+        {
+            var groupOnPage = GetEndpointGroupOnDetailsPage().Text;
+            var nameOnPage = GetEndpointNameOnDetailsPage().Text;
+            var tagsOnPage = GetTagsOnDetailsPage();
+
+            Assert.True(string.Equals(_endpointGroup, groupOnPage, StringComparison.CurrentCultureIgnoreCase));
+            Assert.True(string.Equals(_endpointName, nameOnPage, StringComparison.CurrentCultureIgnoreCase));
+            Assert.True(_edpointTags.OrderBy(m => m).SequenceEqual(tagsOnPage.OrderBy(t => t)));
+        }
+
+        public void And_elements_are_rendered()
```

Comment:
you can write here When_elements_are_rendered(), so that the method will be more explicit what is going on here and lightbbd will be smart enough to change it to and (see [steps autogrouping feature](https://github.com/Suremaker/LightBDD/wiki/03-Scenario-Steps-Definition#steps-auto-grouping))